Noted that single-user org. can delete (#2997)
influxdata/docs-v2: content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md

## v1.8.8 [unreleased]

> Due to encountering several issues with the build dependency of v.1.8.8, this version will not be released.  

Reviewer: ```suggestion
Due to encountering several issues with build dependencies in v.1.8.8, this version will not be released.  
```
